spring-boot-project/spring-boot-docs/src/docs/antora/modules/how-to/pages/data-initialization.adoc

```diff
@@ -98,6 +98,7 @@ Spring Boot supports two higher-level migration tools: https://flywaydb.org/[Fly
 === Execute Flyway Database Migrations on Startup
 
 To automatically run Flyway database migrations on startup, add the `org.flywaydb:flyway-core` to your classpath.
+For some databases, a https://search.maven.org/search?q=g:org.flywaydb%20a:flyway-database-*[database-specific module] needs to be added additionally, e.g. `org.flywaydb:flyway-database-postgresql` for PostgreSQL.
```

Comment:
This wildcard search doesn't work for me and reports 0 results. I think we may have to link to https://documentation.red-gate.com/flyway/flyway-cli-and-api/supported-databases instead. There are also some DB-specific modules that are not named `flyway-database-*` such as the [module for MySQL](https://documentation.red-gate.com/flyway/flyway-cli-and-api/supported-databases/mysql).